jobs/ml-ops/inference/main.py

```diff
@@ -0,0 +1,66 @@
+from fastapi import FastAPI
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.metrics import RocCurveDisplay
+import pickle
+import boto3
+import pandas
+import os
+
+import data
+
+classifier = RandomForestClassifier()
+
+app = FastAPI()
+
+MODEL_FILE = "classifier.pkl"
+
+
+class ClassifierLoader(object):
+    _classifier = None
+
+    @classmethod
+    def load(cls, force=False):
+        if force or cls._classifier is None:
+            access_key = os.environ["ACCESS_KEY"]
+            secret_key = os.environ["SECRET_KEY"]
+            region_name = os.environ["REGION"]
+
+            bucket_name = os.environ["S3_BUCKET_NAME"]
+            s3_url = os.environ["S3_URL"]
+
+            s3 = boto3.client(
+                "s3",
+                region_name=region_name,
+                endpoint_url=s3_url,
+                aws_access_key_id=access_key,
+                aws_secret_access_key=secret_key,
+            )
+
+            s3.download_file(bucket_name, MODEL_FILE, MODEL_FILE)
+
+            with open(MODEL_FILE, "rb") as fh:
+                cls._classifier = pickle.load(fh)
+
+        return cls._classifier
+
+
+@app.get("/load")
```

Comment:
nitpick: for actions, I prefer using post methods. Reference: https://restful-api-design.readthedocs.io/en/latest/methods.html#actions

```suggestion
@app.post("/load")
```